topology-manager/src/main/java/org/apache/aries/rsa/topologymanager/importer/TopologyManagerImport.java

```diff
@@ -66,7 +67,7 @@
     public TopologyManagerImport(BundleContext bc) {
         this.rsaSet = new HashSet<>();
         bctx = bc;
-        execService = new ThreadPoolExecutor(5, 10, 50, TimeUnit.SECONDS, new LinkedBlockingQueue<Runnable>());
+        execService = new ThreadPoolExecutor(5, 10, 50, TimeUnit.SECONDS, new LinkedBlockingQueue<Runnable>(), new NamedThreadFactory("TopologyManagerImporter-%d"));
```

Comment:
Maybe use pattern with `this.class.simpleName()` instead of hardcoding class name ins string?

Reply:
Ok!